lms/djangoapps/ccx/api/v0/serializers.py

```diff
@@ -31,6 +31,7 @@ class Meta:
             "due",
             "max_students_allowed",
             "course_modules",
+            "other_course_settings",
```

Comment:
Please, add this field to the serializer attributes. @JuanDavidBuitrago

Reply:
Done